week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -15,8 +23,20 @@ Complete the function to check if the variable `num` satisfies the following req
 Tip: use logical operators
 */
 
-function validate(num) {}
-
+function validate(num) {
+  if ( typeof num === 'number' && num <= 0 && num % 2 === 0) {
```

Comment:
This is almost correct, but I think there's a small mistake in one of your comparisons. The second one is supposed to check that the number is less than or equal to 100, not 0 like you have here.